installer/bundle/bundle_skel_Linux.sh

```diff
@@ -620,6 +621,10 @@ case "$installMode" in
             if [ $? -eq 0 ]; then
                 shouldInstall_omi
                 pkg_upd_list $OMI_PKG omi $?
+            else
+                pkg_add_list $OMI_PKG omi
+                pkg_add ${ADD_PKG_QUEUE}
+                OMI_EXIT_STATUS=$?
```

Comment:
Thanks for the pointer. But I did not get the problem.

> you're mixing ADD and UPDATE operations, and thus forcing the components to be updated independently.

In the current scenario, as far as the linux package manager is concerned, there are two packages - scx and omi.
One needs to be installed (omi). Other has to be upgraded (scx).

oms agent is doing the following

              shouldInstall_omsagent
             pkg_upd_list $OMS_PKG omsagent $?

In the current case, bundle_skel_Linux.sh and oms-agent script will have exactly same outcome.
And the earlier version of this script, did exactly like this 
Here is the [link](https://github.com/Microsoft/SCXcore/blob/b95b9350b0341aa51c67053045a5fa212ae092e0/installer/bundle/bundle_skel.sh#L748_L759) to the relevant section of the code.
As I understand, thorough upgrade testing was done on this script and it has always passed till this change.